src/components/title-blog-index/title-blog-index.jsx

```diff
@@ -12,19 +12,27 @@ const TitleBlogIndex = ({ data, refIndexTitles, disappearIndex }) => {
   const [selectLink, setSelectLink] = useState('');
   const [isTopArrow, setIsTopArrow] = useState(false);
   const [isBottomArrow, setIsBottomArrow] = useState(false);
-  const subContainerRef = useRef(null);
-  const viewContainerRef = useRef(null);
+  const linksContainerRef = useRef(null);
+  const indexContainerRef = useRef(null);
+  const [indexHeight, setIndexHeight] = useState(0);
+  const [linksHeight, setLinksHeight] = useState(0);
```

Comment:
you always use and set these two states together. Why keep them separate?

Reply:
But you don't need them separately. You only use the difference between them which can be just one state